src/main/java/liuni/dagger/TwitterComponent.java

```diff
@@ -0,0 +1,9 @@
+package liuni.dagger;
```

Comment:
if you're going to have different components/modules, just put it in a single class so there is less overhead of injection class creation

Reply:
Done... Dagger magic is just a black box.